### PR TITLE
Adding Helium Geek to Community Tools

### PR DESCRIPTION
**Pull request by riobah**
Also adding a new type for "Android"

### CODE REVIEW

File: data/communitytools.js [hunk 1]
```diff
@@ -24,6 +24,11 @@ export const types = {
     foregroundColor: '#32C48D',
     backgroundColor: '#ccf1e8',
   },
+  android: {
+    label: 'Android',
+    foregroundColor: '#6486cb',
+    backgroundColor: '#222E46',
```
Comment (danielcolinjames) [inline data/communitytools.js]:
```suggestion
    backgroundColor: '#f8d4d9',
```

File: data/communitytools.js [hunk 2]
```diff
@@ -24,6 +24,11 @@ export const types = {
     foregroundColor: '#32C48D',
     backgroundColor: '#ccf1e8',
   },
+  android: {
+    label: 'Android',
+    foregroundColor: '#6486cb',
```
Comment (danielcolinjames) [inline data/communitytools.js]:
```suggestion
    foregroundColor: '#FF4747',
```

**Review by danielcolinjames — CHANGES_REQUESTED** [review]
hey @riobah, thanks for submitting this!

thanks for adding an android type too. I think the colours look slightly off, could we try these instead? (suggested in comments)

**Comment by riobah** [comment]
Thank you for the review. Just made the changes.

**Review by danielcolinjames — APPROVED** [review]

**Pull request merged**